tests/dags/test_dag_serialization.py

```diff
@@ -244,7 +244,6 @@ def test_serialization(self):
             # Verify JSON schema of serialized operators.
```

Comment:
Should remove the hardcoding of dag names on the line 36

https://github.com/apache/airflow/pull/5743/files#diff-73d9177194b24994293316f69a960ec1R36

Reply:
I can work on this one.

Reply:
Done